project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,685 @@
+      - *EnergyStar_window_southern
+      - *infiltration_30pct_reduction_windows
+      - *infiltration_15pct_reduction_windows_energystar_northern
+      - *infiltration_15pct_reduction_windows_energystar_north_central
+      - *infiltration_15pct_reduction_windows_energystar_south_central
+      - *infiltration_15pct_reduction_windows_energystar_southern
+      #1” XPS (R-5/in) Exterior wall insulation for dwelling units with wall insulation of less than R-19  
+      - &exterior_wall_insulation
+        option: Insulation Sheathing|R-5
+        apply_logic:
+          - or:
+            - Insulation Wall|Wood Stud, Uninsulated
+            - Insulation Wall|Wood Stud, R-7
+            - Insulation Wall|Wood Stud, R-11
+            - Insulation Wall|Wood Stud, R-13
+            - Insulation Wall|Wood Stud, R-15
+            - Insulation Wall|Wood Stud, Uninsulated, R-5 Sheathing
+            - Insulation Wall|Steel Stud, Uninsulated
+            - Insulation Wall|Steel Stud, R-13
+        costs:
+          - value: 0.0443 # TBD
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value) # TBD
+        lifetime: 60
+      - &infiltration_IECC2021_with_ERV_added
+        option: Infiltration|3 ACH50
+        apply_logic: &infiltration_higher_than_3ACH50
+          - or:
+            - Infiltration|50 ACH50
+            - Infiltration|40 ACH50
+            - Infiltration|30 ACH50
+            - Infiltration|25 ACH50
+            - Infiltration|20 ACH50
+            - Infiltration|18.5 ACH50
+            - Infiltration|15 ACH50
+            - Infiltration|11.25 ACH50
+            - Infiltration|10 ACH50
+            - Infiltration|8 ACH50
+            - Infiltration|7.5 ACH50
+            - Infiltration|7 ACH50
+            - Infiltration|7 ACH50, 0.5 Shelter Coefficient
+            - Infiltration|6 ACH50
+            - Infiltration|5.25 ACH50
+            - Infiltration|5 ACH50
+            - Infiltration|4.5 ACH50
+            - Infiltration|4 ACH50
+            - Infiltration|3.75 ACH50
+        costs:
+          - value: 0.50 # To be decided
+                        # $0.50/ft2 is from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # Interpolated between 20% and 40% reductions to get $0.44/ft2,
+                        # then used 1.14 multiplier to inflate from Jan 2019 to Mar 2022
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &infiltration_IECC2021_add_ERV
+        option: Mechanical Ventilation|ERV, 72%
+        apply_logic:
+        - and:
+            - or:
+              - Mechanical Ventilation|None
+              - Mechanical Ventilation|Exhaust
+              - Mechanical Ventilation|Supply
+            - or: *infiltration_higher_than_3ACH50
```

Comment:
Let's confirm with @trynthink , but for now (small run QC) lets do this regardless of ACH50, even <3 ACH50.

Reply:
Done